Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -141,5 +155,82 @@ private void OnDestroy()
                 Destroy(debugSphere.gameObject);
             }
         }
+
+        private static bool IsObjectPartOfGrabbable(GameObject targetObject, BaseNearInteractionGrabbable grabbable)
+        {
+            return targetObject != null && grabbable != null &&
+                (targetObject == grabbable.gameObject ||
+                // Descendant game objects are grabbable as well.
+                (targetObject.transform != null && grabbable.gameObject.transform != null &&
+                targetObject.transform.IsChildOf(grabbable.gameObject.transform)));
+        }
+
+        /// <inheritdoc />
+        public override void OnInputDown(InputEventData eventData)
+        {
+            if (Result?.CurrentPointerTarget != null && TryGetNearGraspPoint(out Vector3 pointerPosition))
+            {
+                if (currentGrabbableObjectDown == null)
+                {
+                    // In order to get reliable up/down event behavior, only allow the closest grabbable to be grabbed.
+                    if (IsObjectPartOfGrabbable(Result.CurrentPointerTarget, closestProximityGrabbable))
+                    {
+                        currentGrabbableObjectDown = Result.CurrentPointerTarget;
+
+                        //For now we're only generating pointer events
+                        if (Application.isEditor)
```

Comment:
This is not a reliable solution, we should fix the input simulation instead if there is a discrepancy with hand tracking. I'm trying to understand how these actions work, but don't have a good understanding yet.

Reply:
This is a weird one, the GrabPointer is set to "Grip Press" - which on device works great. In editor, the "pinch" gesture only fires "Select". I don't know enough about input simulation to make that fix so I just stuck this in here. PokePointer doesn't even use the base.OnInputDown call so I figured it was okay.

Reply:
force reverted to go down a simpler route.